versions/2016-10-31.md

```diff
@@ -796,6 +797,15 @@ DeploymentPreference:
     # Validation Lambda functions that are run before & after traffic shifting
     PreTraffic: !Ref PreTrafficLambdaFunction
     PostTraffic: !Ref PostTrafficLambdaFunction
+  TriggerConfigurations:
+    # A list of trigger configurations you want to associate with the deployment group. Used to notify an SNS topic on 
+    # lifecycle events.
+    - TriggerEvents:
+      # A list of events to trigger on.
+      - DeploymentSuccess
+      - DeploymentFailure
+      TriggerName: TestTrigger
+      TriggerTargetArn: !Ref MySNSTopic  # SNS Topic must be created in the same template
```

Comment:
The comment is incorrect, right? The SNS topic doesn't have to be created in the same template for this to work since we're just passing it right through to the `AWS::CodeDeploy::DeploymentGroup` resource. I'd suggest removing the comment as it will cause confusion.

Reply:
You're right, not sure what my thought process was there. Removed the offending comment!